### PR TITLE
update zh-cn translation

### PR DESCRIPTION
**Pull request by hqwrong**
Hope I can help the community in this way :)

### CODE REVIEW

File: translations/monero-core_zh-cn.ts [hunk 1]
```diff
@@ -814,32 +814,32 @@
...
         <location filename="../pages/Settings.qml" line="120"/>
         <source>Show seed &amp; keys</source>
-        <translation type="unfinished"></translation>
+        <translation>显示密语和密钥</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="163"/>
         <source>Rescan wallet balance</source>
-        <translation type="unfinished"></translation>
+        <translation> 重新扫描钱包余额</translation>
```
Comment (erciccione) [inline translations/monero-core_zh-cn.ts]: Please remove the space at the beginning of the string

**Comment by hqwrong** [comment]
I fixed it and made a whole new one-commit [PR:](https://github.com/monero-project/monero-gui/pull/1080)

**Comment by Maxithi** [comment]
In the future you can just push to the same branch to update this PR rather than making a new one.

**Comment by hqwrong** [comment]
Oh, okay.

**Pull request closed without merging**